samples/sensor/grow_r502a/README.rst

```diff
@@ -0,0 +1,56 @@
+.. _grow_r502a:
+
+GROW_R502A Fingerprint Sensor
+#############################
+
+Overview
+********
+
+This sensor sample shows template count inside fingerprint sensor and do the following,
+Finger enrollment ,fingerprint Matching and deleting. Sensor LED will blink accordingly.
+
+enrollment and matching should work only when trigger mode is set.
```

Comment:
Please fix this up.

Reply:
I'm sorry, but I still can't parse this section; can you please try to fix the grammar?

Reply:
This is the elloborate content I am trying to say. are you expecting like this or do I have to explain the functions of the sensor here instead of explaining the sample working?

```
This sensor sample shows the number of fingerprints (template count) inside the sensor.
This sample adds a new fingerprint (enrollment) when SENSOR_ATTR_ADD is set, matches
Your placed finger with an existing fingerprint inside sensor and shows the matching ID and
its confidence when SENSOR_ATTR_FIND is get, deletes stored fingerprint using its 
ID when SENSOR_ATTR_DEL is set.

Tri-colour LED around fingerprint sensor hardware will blink or flash for each
Attributes,for successful addition and deletion of fingerprint blue LED will blink
Three times, for successful matching purple LED will flash. for all other error cases
Red LED will blink.

Fingeprint adding and matching functionalities only works when SENSOR_TRIG_TOUCH is set.
```

Reply:
@mbolivar-nordic  ping!

Reply:
I'm saying there are typos and grammar problems there, and in the comment here: https://github.com/zephyrproject-rtos/zephyr/pull/40394#discussion_r826601128. They are making it hard for me t understand this. Can you please fix the English?

Reply:
```
This sample has the below functionalities:
1. Shows the number of fingerprints stored in the sensor.
2. When SENSOR_ATTR_ADD is set then it adds a new fingerprint to the sensor.
3. When SENSOR_ATTR_FIND is set then it tries to find a match for the input fingerprint. On finding a match it returns ID and confidence.
4. When SENSOR_ATTR_DEL is set then it deletes a fingerprint from the sensor.
Note: Fingerprint add & delete functionalities work only when SENSOR_TRIG_TOUCH is set.
Tricolored LED in the sensor hardware will, flash on the below conditions:
1. On successful addition or deletion of fingerprint it will flash in blue three times.
2. On finding a match for the input fingerprint it will flash in purple.
3. In all other cases it will flash in red.
```

@mbolivar-nordic  Hope this overview is understandable and also having no grammatical mistakes.